scenarios/appsec/test_customconf.py

```diff
@@ -37,7 +37,7 @@ def test_c05(self):
         stdout.assert_presence(r"AppSec could not read the rule file .* as it was invalid: .*", level="CRITICAL")
 
 
-@released(java="0.93.0", nodejs="2.4.1", php_appsec="0.3.0", ruby="?")
+@released(java="0.93.0", php_appsec="0.3.0", ruby="?")
```

Comment:
I do not understand, you're saying here that this feature is working for all nodejs tracer version Oo

Reply:
a `?`